### PR TITLE
Bump prosemirror-test-builder, prosemirror-schema-basic and prosemirror-schema-list

### PR DESCRIPTION
**Pull request by nlam-atlassian**
Bumping the test modules because some of the new tests depend on it.
There are lots more tests to convert, but I've just converted a few more in this PR to make sure that we have what we need in the test modules.

### CODE REVIEW

File: test-builder/README.md [hunk 1]
```diff
@@ -89,3 +89,11 @@ Check the latest package at Maven central on: https://packages.atlassian.com/mav
 ```kotlin
 implementation("com.atlassian.prosemirror:test-builder:1.0.2")
 ```
+### Versioning
+- Implemented changes to match version:
```
Comment (douglasmarques) [inline test-builder/README.md:93]: It would be nice to add this versioning section in each module with the respective web version.

**Review by douglasmarques — APPROVED** [review]

File: test-builder/src/commonMain/kotlin/com/atlassian/prosemirror/testbuilder/schema-list.ts [hunk 2]
```diff
@@ -155,6 +155,19 @@ export function splitListItem(itemType: NodeType, itemAttrs?: Attrs): Command {
   }
 }
 
+/// Acts like [`splitListItem`](#schema-list.splitListItem), but
+/// without resetting the set of active marks at the cursor.
+export function splitListItemKeepMarks(itemType: NodeType, itemAttrs?: Attrs): Command {
```
Comment (tchernykh) [inline test-builder/src/commonMain/kotlin/com/atlassian/prosemirror/testbuilder/schema-list.ts:160]: Was this method ported to Kotlin code? I can't find it.

File: test-builder/README.md (hunk 1 above)
Comment (nlam-atlassian) [reply test-builder/README.md:93]: Yes! This is my plan. I've done this for the modules that I have already updated. And will slowly go through the other modules and do the same - once I can verify what the version is!

File: test-builder/src/commonMain/kotlin/com/atlassian/prosemirror/testbuilder/schema-list.ts (hunk 2 above)
Comment (douglasmarques) [reply test-builder/src/commonMain/kotlin/com/atlassian/prosemirror/testbuilder/schema-list.ts:160]: https://github.com/ProseMirror/prosemirror-schema-list/blob/master/src/schema-list.ts

Comment (nlam-atlassian) [reply test-builder/src/commonMain/kotlin/com/atlassian/prosemirror/testbuilder/schema-list.ts:160]: There were a few functions in this file that were not originally converted either. I guess we don't need them yet. We can convert them when we do.

**Pull request merged**